### ISSUE TITLE
NVDA driver: when allow skipping while say all option is checked, RhVoice does not read text

### ISSUE THREAD

**Issue by burakyuksek** [RHVoice/RHVoice#860]
1. Turn on the option mentioned in the issue title from keyboard settings
2. Choose bdl as voice
3. Go to some wepage in chrome
4. press say all command
5. Press down arrow while say all is in progress.
RHVoice will stop speaking.
It should not stop speaking, instead it shoudl speak the current line NVDA has skipped to.

**Comment by zstanecic** [comment]
Hi,

Can you post logs?

**Comment by burakyuksek** [comment]
[nvda.log](https://github.com/RHVoice/RHVoice/files/14640069/nvda.log)

**Comment by burakyuksek** [comment]
@zstanecic In case you missed, at first I sent the log but forgot to mention you. Then I have edited the post to mention you.

**Comment by zstanecic** [comment]
I don't see logs, unfortunatelly, and github never notified  for a such case.

**Comment by burakyuksek** [comment]
@zstanecic 
[nvda.log](https://github.com/user-attachments/files/16400247/nvda.log)
